crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -730,6 +730,34 @@ impl<'w> EntityMut<'w> {
         unsafe { component_ids.fetch_mut(self.0) }
     }
 
+    /// Returns [untyped mutable reference](MutUntyped) to component for
+    /// the current entity, based on the given [`ComponentId`].
+    ///
+    /// Unlike [`EntityMut::get_mut_by_id`], this method borrows &self instead of
+    /// mut &self, allowing the caller to access multiple components simultaneously.
+    ///
+    /// # Errors
+    ///
+    /// - Returns [`EntityComponentError::MissingComponent`] if the entity does
+    ///   not have a component.
+    ///
+    /// # Safety
+    /// It is the callers responsibility to ensure that
+    /// - Exclusive access to all components of this entity,
```

Comment:
shouldn't the safety comments here be the same as on https://dev-docs.bevyengine.org/bevy/ecs/world/unsafe_world_cell/struct.UnsafeEntityCell.html#method.get_mut_by_id?